python-package/lightgbm/sklearn.py

```diff
@@ -719,10 +719,9 @@ def predict(self, X, raw_score=False, start_iteration=0, num_iteration=None,
             X = _LGBMCheckArray(X, accept_sparse=True, force_all_finite=False)
         n_features = X.shape[1]
         if self._n_features != n_features:
-            raise ValueError("Number of features of the model must "
-                             "match the input. Model n_features_ is %s and "
-                             "input n_features is %s "
-                             % (self._n_features, n_features))
+            raise ValueError(f"Number of features of the model must " \
+                             f"match the input. Model n_features_ is {self._n_features} and " \
+                             f"input n_features is {n_features} ")
```

Comment:
```suggestion
            raise ValueError("Number of features of the model must "
                             f"match the input. Model n_features_ is {self._n_features} and "
                             f"input n_features is {n_features} ")
```

Can you please remove these backslashes? They are not necessary. This is why the linting job is complaining. Also, the first string (which doesn't have any templating stuff with `{}`) does not need to be an f-string.

https://github.com/microsoft/LightGBM/pull/4188/checks?check_run_id=2362245578

```text
Linting Python code
./python-package/lightgbm/sklearn.py:722:71: E502 the backslash is redundant between brackets
./python-package/lightgbm/sklearn.py:723:95: E502 the backslash is redundant between brackets
./python-package/lightgbm/sklearn.py:922:75: E502 the backslash is redundant between brackets
./python-package/lightgbm/sklearn.py:923:90: E502 the backslash is redundant between brackets
```

In case you're not familiar with it, multiple consecutive string literals within parentheses will be interpreted as a single string by Python.

```python
one_line = "it is cloudy"
multi_line = (
    "it "
    "is "
    "cloudy"
)

assert one_line == multi_line
print(multi_line)
```

Reply:
I was trying to use a multi-line f-string, but I do see where it wasn't required.